spec/std/base64_spec.cr

```diff
@@ -61,6 +61,49 @@ describe "Base64" do
     assert_prints base64_strict_encode(array), "AQEBAQE="
   end
 
+  context "encoding an IO" do
+    encode_buffer = IO::Memory.new
+    decode_buffer = IO::Memory.new
+
+    before_each do
+      # Clear out each buffer before each spec
+      encode_buffer = IO::Memory.new
+      decode_buffer = IO::Memory.new
+    end
```

Comment:
**question:** Why do you define these variables outside the examples?
This could be useful if you're planning to reuse the buffers (with `IO::Memory#clear`), but with fresh instances it would be easier to just push the initialization into the test block.

Reply:
> Why do you define these variables outside the examples?

It was how I wrote the specs in my app because, when I wrote it, I just wanted to make it work with sufficient confidence. It was a pure copy/paste from my app into this PR. I was trying to push the PR in between a bunch of other things during a very busy evening. I didn't have time to polish it up.